corehq/apps/userreports/tasks.py

```diff
@@ -10,7 +10,7 @@
 from dimagi.utils.couch.database import iter_docs
 
 
-@task(queue='background_queue', ignore_result=True)
+@task(queue='background_queue', ignore_result=True, acks_late=True)
```

Comment:
Thanks for the heads-up. This is indeed a long-running task. Do you think anything should be done to mitigate the risk of a message broker change breaking this?